implants/lib/eldritch/src/file/template_impl.rs

```diff
@@ -21,8 +21,10 @@ pub fn template(
     autoescape: bool,
 ) -> Result<()> {
     let context = build_context(args)?;
-    let template_content = fs::read_to_string(template_path)?;
-    let res_content = Tera::one_off(template_content.as_str(), &context, autoescape)?;
+    let data = fs::read(template_path.clone())?;
```

Comment:
whats the difference here?

Reply:
Won't fail when reading non-UTF8 characters like a binary file.

Reply:
ah, i see.